core/conversion/converters/converter_util.cpp

```diff
@@ -199,6 +199,60 @@ nvinfer1::ITensor* tensor_to_const(ConversionCtx* ctx, at::Tensor t, const std::
   return out;
 }
 
+nvinfer1::ITensor* toITensor(ConversionCtx* ctx, const torch::jit::Node* n, at::Tensor* input) {
```

Comment:
Dont we already have something for this? https://github.com/pytorch/TensorRT/blob/2585b773dfa6da64698b6505b43cbb4dc7e9c037/core/conversion/converters/converter_util.h#L51